### ISSUE TITLE
Default the content of network to '*'

### ISSUE THREAD

**Issue by gmarty** [arcturus/node-appcache-generator#2]
If there's no content passed to the network section, it's probably a good idea to use `*` as the default value:

```
NETWORK:
*
```

**Comment by emberdart** [comment]
I agree. As an easier workaround than providing a file, I've gone with:

echo "*" | node-appcache -n /dev/stdin -d .

for now.
